pkg/server/keyclaim.go

```diff
@@ -85,6 +87,18 @@ func (s *keyClaimServlet) newKeyClaim(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	event := persistence.Event{
+		Originator: originator,
+		DeviceType: persistence.Server,
+		Identifier: persistence.OTKGenerated,
+		Date:  time.Now(),
+		Count: 1,
+	}
+	if err := s.db.SaveEvent(event); err != nil {
+		// We don't necessarily want to crash if we were unable to log a metric
+		log(nil, err).Warn(fmt.Sprintf("Unable to log event: %#v\n", event))
```

Comment:
Again maybe we want to standardize this with `WithFields`

Reply:
Done